### PR TITLE
Add cluster name to credentials information

### PR DESCRIPTION
**Pull request by TheBigLee**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Categorize the PR by setting a good title and adding one of the labels:
  `bug`, `enhancement`, `documentation`, `change`, `breaking`,
  as they show up in the changelog
- [x] Update the documentation.
- [x] Update tests.
- [ ] Link this PR to related issues.

### CODE REVIEW

File: pkg/crossplane/service_mariadb_database.go [hunk 1]
```diff
@@ -243,7 +246,7 @@ func mapMariadbEndpoint(data map[string][]byte) (*Endpoint, error) {
 	}, nil
 }
 
-func createCredentials(endpoint *Endpoint, username, password, database, metricsPort string) Credentials {
+func createCredentials(endpoint *Endpoint, username, password, database, cn string, metricsEnabled bool, md string) Credentials {
```
Comment (glrf) [inline pkg/crossplane/service_mariadb_database.go:249]: Minor nitpick: `cn` and `md` cloud be a bit more self explanatory. I.e. `clusterName` and `metricsDomain`

File: pkg/crossplane/service_redis.go [hunk 2]
```diff
@@ -44,6 +44,12 @@ func (rsb RedisServiceBinder) Deprovisionable(ctx context.Context) error {
 	return nil
 }
 
+// GetClusterName returns the cluster name of the instance
+func (instance Instance) GetClusterName() string {
+	labels := instance.Composite.GetLabels()
+	return labels["service.syn.tools/cluster"]
+}
```
Comment (glrf) [inline pkg/crossplane/service_redis.go:51]: This method should be in `instance.go` not in `service_redis.go`.

**Review by glrf — APPROVED** [review]
LGTM.

Feel free to ignore code climate for this. `ReadConfig` is a bit long, but that's IMO totally fine. I think the repo is setup weirdly that it requires code climate to pass. Sadly I don't have permissions on this repo to fix this.

Also I have no idea what went wrong with the e2e report..

File: pkg/crossplane/service_redis.go (hunk 2 above)
Comment (TheBigLee) [reply pkg/crossplane/service_redis.go]: Moved the method

**Review by mhutter — COMMENTED** [review]
Is this on purpose that many tests have been removed?

**Comment by TheBigLee** [comment]
> Is this on purpose that many tests have been removed?

Yes. Those tests are from an old implementation for the metrics endpoints that in the end didn't work out (see: #51 )

**Review by mhutter — APPROVED** [review]

**Pull request merged**